tests/ethpm/test_deployments.py

```diff
@@ -1,4 +1,4 @@
-from eth_utils import to_bytes
+from eth_utils import to_bytes, to_canonical_address
```

Comment:
I keep forgetting this repo uses `black` when I see import statements not ordered as expected.  At first it threw me off but now I'm starting to like it (not worrying about this stuff and not having to maintain the weird subjective rulesets in my head)